src/ServicePulse.Host/app/js/services/service.formatter.js

```diff
@@ -33,6 +33,24 @@
             }
         }
 
+        function formatDate(value) {
+            return value ? moment(value).format("MMMM DD, YYYY") : undefined;
```

Comment:
The screenshot from @sergioc shows MM/dd/YYYY, should we change to match?

Reply:
That format is a bit US-centric, isn't it? I thought Europe usually used dd/MM/yyyy.

What I _really_ wanted was to use the browser's locale but that seems to be tougher than it looks with moment. Either way, I'm a bit paranoid about the MM/dd and dd/MM formats so I tend to default to something more explicit.